lib/mengpaneel/tracker.rb

```diff
@@ -90,6 +90,15 @@ def update(message)
 
         super(message)
       end
+
+      # mixpanel-ruby only handles hash, whereas Javascript handles string and hash.
+      def increment(*args)
+        if args.first.is_a?(Hash)
+          super(tracker.distinct_id, *args)
```

Comment:
Ok, now I know what is this intended for. However, it will cause a problem when distinct_id equals to key of property name, for example if you are using a username as distinct_id, and the username is *login*, `mixpanel.people.increment('login')` will fail.

Instead, I'd like to create a new class like `class DistinctId < String; end` and encapsulate distinct_id in `#identify`. You can still use `@distinct_id` as a normal string value so it doesn't break current behaviour, but can know whether it is distinct_id or not via `args.first.is_a?(DistinctId)`. How do you think?